README.md

```diff
@@ -16,7 +16,7 @@ npm run dev
 
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the built project.
 
-## Stlying
+## Styling
```

Comment:
Please change this this thing. Not according to the guidelines.

Reply:
it is